package.json

```diff
@@ -123,5 +123,5 @@
       "tag": true
     }
   },
-  "version": "0.2.3"
+  "version": "0.2.4"
```

Comment:
How come there is a release bump in this PR? I would kindly ask you to let us review the PR first and then use your powers to publish. It is not a big PR so you are forgiven this time ;)